test/pagespeed_test.conf.template

```diff
@@ -616,6 +616,13 @@ http {
   }
 
   server {
+    listen @@SECONDARY_PORT@@;
+    server_name date.example.com;
+    pagespeed FileCachePath "@@FILE_CACHE@@";
+    add_header "Date" "Date: Fri, 16 Oct 2009 23:05:07 GMT";
```

Comment:
Note: This `add_header` line looks wrong, which is already fixed on trunk-tracking via https://github.com/pagespeed/ngx_pagespeed/commit/a8141eadd3e22a66fcbc5dacecf97b9669c3a5ef